### DISCUSSION TITLE
Update Album Thumbnails Progress Bugs post v 20240226 update

### DISCUSSION THREAD

**Discussion by ccaper** [rern/rAudio#1611] [Fixed Issues]
Very minor, but thought I'd share.  I first noticed the issue after updating to 20240226 (I was on 20240212).  I have confirmed this issue does not exist in 20240212.

When I update thumbnail art, the progress status has a couple of display issues.
1. The total count in the progress count always shows "1" instead of the total  number of albums.  For example, when on the 5 album of 100 total albums, the progress shows "5/1"
2. The percentage complete is wrong.  For example, on the first album updated, it shows 100%, 2nd album 200%, 3rd album 300%.  I believe it is related to problem in 1 mentioned above on computing the math for the percentage (the total count is always 1).

Updating album art does complete, and does update album art correctly, so this is very minor.  As always, thanks for the amazing work on this app!

**Comment by kw-sudo** [comment]
+1 here (in 20240312)

![raudio](https://github.com/rern/rAudio/assets/104136223/470c320c-8a1a-49fb-b523-7f8dc5507679)

Regards,
Oliver

edit: correct version

**Reply by ccaper** [reply]
This is fixed as of 20240315 update.  Thank you!
